numerics/pid.hpp

```diff
@@ -0,0 +1,49 @@
+﻿
+#pragma once
+
+#include <deque>
+
+#include "quantities/named_quantities.hpp"
+#include "quantities/quantities.hpp"
+
+namespace principia {
+namespace numerics {
+namespace internal_pid {
+
+using quantities::Inverse;
+using quantities::Time;
+
+template<typename Value, int horizon, int finite_difference_order>
+class PID {
+public:
+  static_assert(finite_difference_order <= horizon);
+
+  PID(double kp, Inverse<Time> const& ki, Time const& kd);
+
+  // Clears the state of the PID.
+  void Clear();
+
+  // Adds the error between the two values to the state of the PID and
+  // returns an apparent value derived from the actual value and the control
+  // variable.
+  Value ComputeValue(Value const& apparent,
+                     Value const& actual,
```

Comment:
Is `apparent` and `actual` the standard terminology here?

Reply:
The standard terminology is "process variable/value/parameter" and "setpoint/set point/set-point".  Changing accordingly.